docs/custom_virtualenvs.md

```diff
@@ -47,7 +54,7 @@ can be found using `pip freeze`:
 
 One important item to keep in mind is that in a clustered awx installation,
 you need to ensure that the same custom virtualenv exists on _every_ local file
-system at `/var/lib/awx/venv/`.  For container-based deployments, this likely
+system at `/opt/my-envs/`.  For container-based deployments, this likely
 means building these steps into your own custom image building workflow, e.g.,
```

Comment:
Don't you need to then include adding it to `CUSTOM_VENV_PATHS` ?

Reply:
Yep, good point.